src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/OOPTagHelperResolver.cs

```diff
@@ -102,6 +102,7 @@ protected virtual async Task<TagHelperResolutionResult> ResolveTagHelpersOutOfPr
             // This will change in the future to an easier to consume API but for VS RTM this is what we have.
             try
             {
+#pragma warning disable CS0612 // Type or member is obsolete
```

Comment:
Could we just tack what type or member is obsolete on to the end here?

Reply:
Sure